### PR TITLE
Typos and fixes

### PR DESCRIPTION
**Pull request by bentley**
Most of these errors were present in the original scans, and many other errors besides—understandable given the sources are classic scifi mags. I did my best to limit to only the most unambiguous or offensive typos.

b6c9723bab20aa70a292c69cbee57b13aefa25d0 bears special mention—it appears to contain two drafts of the same paragraph.

### CODE REVIEW

File: src/epub/text/lazarus-come-forth.xhtml [hunk 1]
```diff
@@ -53,7 +53,7 @@
 			<p>“Maybe⁠—just maybe⁠—we <em>have</em> got the weapon,” he said.</p>
 			<p>His hand trembled.</p>
 			<hr/>
-			<p>The coroner pumps throbbed warmly under the table, while manipulating tendrils darted swiftly, effectively over the dead Scientist’s body. Brandon moved, too, like a machine. In a regular fury he had forced Logan to hurry the body down into the preparations room, inject adrenalin, thermal units, apply the blood pump and accomplish a thousand other demanding and instantaneous tasks.</p>
+			<p>The coroner pumps throbbed warmly under the table, while manipulating tendrils darted swiftly, effectively over the dead Scientist’s body. Brandon moved, too, like a machine. In a regular fury he had forced Logan to hurry the body down into the preparations’ room, inject adrenalin, thermal units, apply the blood pump and accomplish a thousand other demanding and instantaneous tasks.</p>
```
Comment (acabal) [inline src/epub/text/lazarus-come-forth.xhtml]: I don't think this is correct, this change implies the the room belongs to "the preparations", whatever that is? Isn't `preparations room` the name of the room in which preparations occur?

Unless this is some unusual plot point, I'd suggest that it's the other instances of `preparations’` that are incorrect, not this one.

Comment (bentley) [reply src/epub/text/lazarus-come-forth.xhtml]: I agree.

**Comment by bentley** [comment]
Updated to use “preparations room” instead of “preparations’ room.”

**Comment by acabal** [comment]
Thanks!

**Pull request merged**